docs/pages/kubernetes-access/helm/reference/teleport-kube-agent.mdx

```diff
@@ -388,6 +388,41 @@ This can be used for joining a Teleport agent to a Teleport cluster which does n
   One option might be to use Teleport's built-in [ACME support](./teleport-cluster.mdx#acme) or enable [cert-manager support](./teleport-cluster.mdx#highavailabilitycertmanager).
 </Admonition>
 
+## `tls`
+
+### `existingCASecretName`
+
+| Type | Default value |
+| - | - |
+| `string` | `""` |
+
+`tls.existingCASecretName` sets the `SSL_CERT_FILE` environment variable to load a trusted CA or bundle in PEM format into Teleport pods.
```

Comment:
Yes, but not only. `SSL_CERT_FILE` is read by golang's `crypto` package and is used by default to build trust on all TLS clients. This means it can also be used to build trust with a downstream application service signed with a private CA.

Reply:
I added a sentence about this